src/Oscar.jl

```diff
@@ -10,15 +10,39 @@ module Oscar
 import AbstractAlgebra
 import Nemo
 import Hecke
+import Singular
 import Polymake
-#import Singular
+using Markdown
+
+import AbstractAlgebra: @show_name, @show_special
+
+function __init__()
+  println(" -----    -----    -----      -      -----   ")
+  println("|     |  |     |  |     |    | |    |     |  ")
+  println("|     |  |        |         |   |   |     |  ")
+  println("|     |   -----   |        |     |  |-----   ")
+  println("|     |        |  |        |-----|  |   |    ")
+  println("|     |  |     |  |     |  |     |  |    |   ")
+  println(" -----    -----    -----   -     -  -     -  ")
+  println()
+  println("...combining (and extending) Gap, Hecke, Nemo, Polymake and Singular")
```

Comment:
```suggestion
  println("...combining (and extending) GAP, Hecke, Nemo, Polymake and Singular")
```

Reply:
If we mention GAP here, then that's another argument for depending on it and importing it, I guess?